lib/LaTeXML/Common/Font.pm

```diff
@@ -177,16 +186,17 @@ sub decodeFontname {
     if (my $ffam = lookupFontFamily($fam)) { map { $props{$_} = $$ffam{$_} } keys %$ffam; }
     if (my $fser = lookupFontSeries($ser)) { map { $props{$_} = $$fser{$_} } keys %$fser; }
     if (my $fsh  = lookupFontShape($shp))  { map { $props{$_} = $$fsh{$_} } keys %$fsh; }
-    $size = 1 unless $size;    # Yes, also if 0, "" (from regexp)
-    $size = $at if defined $at;
-    $size *= $scaled if defined $scaled;
+    $size        = 1 unless $size;    # Yes, also if 0, "" (from regexp)
+    $size        = $at           if defined $at;
+    $size        = $at * $scaled if defined $scaled;
```

Comment:
Maybe the 3 size assignments here can be compacted to avoid edge cases:
```perl
$size = (defined $at) ? (defined $scaled ? $at * $scaled : $at) : 1;
```

Sample test:
```perl
sub trysize {
  my ($at, $scaled) = @_;
  my $size = (defined $at) ? (defined $scaled ? $at * $scaled : $at) : 1;
  return $size
}

print trysize(undef, undef),"\n";
print trysize(undef, 0.5),"\n";
print trysize(0.5, undef),"\n";
print trysize(0.5, 0.5),"\n";

# output:
# 1
# 1
# 0.5
# 0.25

```

Reply:
That said, the new code will not scale a previously available `$size`, it will only scale with respect to the `$at` value. Is that change intended?

Reply:
Essentially a typo, from backing off a change that wasn't ready. The 2nd line should indeed have been ```$size=$size * $scaled if $scaled;``.  Probably those lines will disappear, or be handled differently in the next PR, so not worth optimizing. commit on the way...